src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -245,7 +390,7 @@ def _yangConvert(val):
             elif 'leafref' in type:
                 vValue = val
             #TODO: find type in sonic-head, as of now, all are enumeration
-            elif 'head:' in type:
+            elif 'stypes:' in type:
```

Comment:
Is this to get the enum define in `sonic-types` yang model?

Reply:
yes this is for definitions in stypes